db/migrations/postgres/000061_add_operation_retry.down.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+ALTER TABLE operations DROP COLUMN retry_id;
```

Comment:
Just a note that you'll need to get a different slot (think currently 70 is next after PRs in the pipe)

Reply:
➡️ ➡️ ➡️ ➡️ ➡️